### ISSUE TITLE
About The ConvLayers

### ISSUE THREAD

**Issue by siwang2011** [smallGum/MLCNN-Multivariate-Time-Series#3]
hello, i find in your paper you use the Conv1D as f
![1624789957(1)](https://user-images.githubusercontent.com/69880884/123541245-11391b00-d776-11eb-8f7d-21f8358167e8.jpg)

but in your code, you use Conv2D, please tell me why you use different ConvLayers?
`
        for i in range(self.n_CNN):
            if i == 0:
                TmpConv = nn.Conv2d(1, self.hidC, kernel_size=(self.kernel_size, idim),
                                    padding=(self.kernel_size // 2, 0))
            else:
                TmpConv = nn.Conv2d(1, self.hidC, kernel_size=(self.kernel_size, self.hidC),
                                    padding=(self.kernel_size // 2, 0))
`

**Comment by smallGum** [comment]
@siwang2011 Hi, in our code, the width of the kernel size is equal to `idim`, that is, the dimension of the multivariate time series. Therefore, the Conv2D with `kernel_size=(self.kernel_size, idim)` performs the same as the Conv1D with `kernel_size=self.kernel_size`. We use Conv2D because it is more directviewing. You can change it into Conv1D if you like.

**Comment by siwang2011** [comment]
thanks for your help.
